app/src/components/common/bar_diagram_probabilities/index.tsx

```diff
@@ -77,11 +77,20 @@ interface Props extends HTMLAttributes<HTMLButtonElement> {
   theme?: any
   winningOutcome: boolean
   withWinningOutcome: boolean
+  displayWinningColor?: boolean
```

Comment:
I would drop this prop, and just show the color when `withWinningOutcome` is true.

Reply:
Done!